tempesta_fw/sock.c

```diff
@@ -208,13 +211,14 @@ ss_turnstile_push(long ticket, SsWork *sw, int cpu)
 	cn->ticket = ticket;
 	memcpy(&cn->sw, sw, sizeof(*sw));
 	spin_lock_bh(&cb->lock);
-	list_add(&cn->list, &cb->head);
+	list_add_tail(&cn->list, &cb->head);
 	cb->size++;
 	if (cb->turn > ticket)
 		cb->turn = ticket;
 	spin_unlock_bh(&cb->lock);
 
```

Comment:
OK, we don't need a barrier after spinlock operation, but having a comment here would be good in case if we remove the spinlock.

Reply:
Done.